fury/interactor.py

```diff
@@ -145,9 +158,49 @@ def _process_event(self, obj, evt):
 
         self.event.reset()  # Event fully processed.
 
+    def _button_clicked(self, button, last_event=-1, before_last_event=-2):
+        if len(self.history) < abs(before_last_event):
+            return False
+
+        if self.history[last_event]["event"] != button + "ButtonReleaseEvent":
+            return False
+
+        if self.history[before_last_event]["event"] \
+                != button + "ButtonPressEvent":
+            return False
+
+        # if self.history[before_last_event]["prop"] \
+        #       != self.history[last_event]["prop"]:
+            # return False
+
+        # dx = self.history[before_last_event]["pos"][0]\
+        #  - self.history[last_event]["pos"][0]
+        # dy = self.history[before_last_event]["pos"][1]\
+        #  - self.history[last_event]["pos"][1]
+        # if (dx**2 + dy**2) > self.click_radius_square:
+            # return False
```

Comment:
We need to decide what we actually want as the double click behavior.

I think it is fine as it is right now, i.e. it is assuming a click radius of 0 because if the user moves their mouse, there will be MouseMoveEvent added to the history between the click events preventing the *DoubleClickEvent to be invoked.

Reply:
Hello @MarcCote , I agree with your points. The current implementation feels natural to me as well. I think we should remove this portion.

Reply:
Great. Let remove it then. @skoudoro, thoughts?

Reply:
I agree with your point too, it is fine as it is. I just tested it and it works well. Thank you @MarcCote and @Nibba2018!
I just think that we need to add them to the UI like we did [here](https://github.com/fury-gl/fury/blob/master/fury/ui.py#L99).
Furthermore, there is a small bug to fix. It seems when you doubleclick on the "background",  there is no object so we need to manage `None` case as you can see below.
<img width="662" alt="Capture d’écran 2020-05-25 à 06 43 43" src="https://user-images.githubusercontent.com/23106443/82778947-25b03300-9e53-11ea-9c62-373f795706ee.png">